web_src/js/vendor/ays.js

```diff
@@ -0,0 +1,182 @@
+// copy of https://github.com/codedance/jquery.AreYouSure/ made to work in strict mode
+// the only changes are the `const` before $fields and $dirtyForms variables
+(function($) {
+
+  $.fn.areYouSure = function(options) {
+
+    var settings = $.extend(
+      {
+        'message' : 'You have unsaved changes!',
+        'dirtyClass' : 'dirty',
```

Comment:
Can we modify var by let, const

Reply:
There are hundrets of lint errors in that file. We could try fixing them at the risk of breaking it.

Reply:
Make sense.